internal/transport/auth_userinfo.go

```diff
@@ -0,0 +1,31 @@
+package transport
+
+import (
+	"net/http"
+
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	"github.com/flightctl/flightctl/internal/auth"
+)
+
+// AuthUserInfo handles OIDC UserInfo requests
```

Comment:
```suggestion
// AuthUserInfo handles UserInfo requests
```
Shouldn't be specific to OIDC providers, but to all of them.

Reply:
fixed